test/SlowTests/Issues/RavenDB-17451.cs

```diff
@@ -0,0 +1,233 @@
+using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests.Server.Replication;
+using Raven.Client.Documents.Subscriptions;
+using Raven.Tests.Core.Utils.Entities;
+using SlowTests.Core.AdminConsole;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_17451 : ReplicationTestBase
+    {
+        public RavenDB_17451(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task CanDisableTcpCompressionViaConfiguration_ReplicationTest(bool disableOnSrc)
+        {
+            var srcServer = GetNewServer(new ServerCreationOptions
+            {
+                RunInMemory = false
+            });
+            var dstServer = GetNewServer(new ServerCreationOptions
+            {
+                RunInMemory = false
+            });
+            using (var srcStore = GetDocumentStore(new Options
+            {
+                Server = srcServer
+            }))
+            using (var dstStore1 = GetDocumentStore(new Options
+            {
+                Server = dstServer
+            }))
+            using (var dstStore2 = GetDocumentStore(new Options
+            {
+                Server = dstServer
+            }))
+            {
+                srcServer.ServerStore.LicenseManager.LicenseStatus.Attributes["tcpDataCompression"] = true;
```

Comment:
Why do you set it to `true`? Our dev / CI licenses have it already enabled so I don't think it's needed